frames/src/main/res/drawable/cko_ic_cross_close.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="14dp"
+    android:height="14dp"
+    android:viewportWidth="14"
+    android:viewportHeight="14">
+  <path
+      android:pathData="M14,1.41L12.59,0L7,5.59L1.41,0L0,1.41L5.59,7L0,12.59L1.41,14L7,8.41L12.59,14L14,12.59L8.41,7L14,1.41Z"
```

Comment:
Question : What is this exactly ? what format is pathData in. Just curious to know.

Reply:
It's a common vector format.
Android Studio provides a possibility to transform svg vector image to xml vector image.